DataFormats/Common/interface/WrapperBase.h

```diff
@@ -54,6 +55,14 @@ namespace edm {
 
     std::shared_ptr<soa::TableExaminerBase> tableExaminer() const { return tableExaminer_(); }
 
+    bool hasMemcpyTraits() const { return hasMemcpyTraits_(); }
+    bool hasMemcpyInit() const { return hasMemcpyInit_(); }
```

Comment:
`hasMemcpyInit()` could be removed, by always calling `dst.memcpyInitialize(src.memcpyParameters())` , and relying on the fact that the implementations of `memcpyInitialize_(...)` and `memcpyParameters_()` in `Wrapper<T>` are almost no-ops if they aren't supported by the type `T`.